scrapers/nd/events.py

```diff
@@ -47,15 +49,16 @@ def consolidate(self):
                 if time_is_earlier(item_time, current_start):
                     self.events[event_key]["event_start_time"] = item_time
 
-            self.events[event_key]["item_keys"].add(item_time)
+            self.events[event_key]["item_keys"].add(item_id)
 
             agenda_item_details = {
                 "description": item["description"],
                 "bill_name": item["bill_name"],
+                "agenda_time": agenda_time,
                 "sub_com": item["sub_com"],
             }
-            self.events[event_key][item_time] = []
-            self.events[event_key][item_time].append(agenda_item_details)
+            self.events[event_key][item_id] = []
```

Comment:
I know this is the code from before, but it seems like:
```
self.events[event_key][item_id] = [agenda_item_details]
```
would do the same thing with less code.

Also, if this particular list only ever gets one item (which seems to be the case when you create an empty list before assigning one item to it...should it even be a list?

An alternative that's less likely to overwrite existing items:
```
if item_id in self.events[event_key]:
    self.events[event_key][item_id].append(agenda_item_details)
else:
    self.events[event_key][item_id] = [agenda_item_details]
```

Reply:
I looked further into it and you're correct, a list isn't required. Simplified with https://github.com/openstates/openstates-scrapers/pull/4523/commits/ba50645163f2eedaf6ab4d5e21828e7ec43d6b44